Proton/Sources/Core/PREditorContentName.h

```diff
@@ -21,25 +21,23 @@
 #ifndef EditorContentName_h
 #define EditorContentName_h
 
-
```

Comment:
`ifdef` doesn't make sense on empty content 😅 It is old way to do pragma once, so expanded to the whole header